python/ray/cloudpickle/cloudpickle_fast.py

```diff
@@ -358,14 +383,6 @@ def _function_setstate(obj, state):
     obj.__globals__.update(obj_globals)
     obj.__globals__["__builtins__"] = __builtins__
 
-    if obj_closure is not None:
```

Comment:
Won't removing this break the pickling if there is a closure?

Reply:
I have moved this code to where the function will be constructed